packages/database/src/migrations/20220808000411-AddDataServiceMappingColumn-modifies-schema.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`ALTER TABLE data_element ADD COLUMN data_service_mapping JSONB DEFAULT '[]'`);
```

Comment:
Not a major thing, and we are planning on removing this in the long run anyway, but I guess I had supposed this would be part of the `config` attribute? Given it's going to be used so rarely, seems it mightn't warrant a column of its own?